client/src/app/site/base/constants/edit-notification-constants.ts

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
This file and the edit-notification.ts, they exist because of the new directive? If so, integrate these files into the directive. These files live in `app/site` which does not make sense if the only usage is in one directive located in `app/shared`.